### ISSUE TITLE
have a bug,redraw the wall,can not add new item

### ISSUE THREAD

**Issue by hsmart** [furnishup/blueprint3d#15]
for new update,it has new bug,if i redraw the wall,then i take the design view ,can't add item in it,or it is only center ,can't move to anywhere.please check it,i want to take part in your project ,but my english do not good . i come from china ,thank you.

**Comment by Ekki** [comment]
I tried to reproduce what you're doing to cause the issue, but failed. Can you please describe it in more detail... e.g. as a step-by-step approach maybe with a few screen shots?

**Comment by Abhishekamol** [comment]
Yes I also face the same problem ,if i redraw the wall, from edit floor plan and make changes in the layout. And after that if i add something through add element then in design, the element is not working proper. Please check. Thank You.

**Comment by hsmart** [comment]
@Abhishekamol 
I Get it! the bug is solved in earlear,and Ekki is update to it.but forgot to fix a position:
`static pointInPolygon(x: number, y: number, corners, startX?: number, startY?: number): boolean {
      //startX = startX || 0;
      //startY = startY || 0;
`
fix it , it works ok.

**Comment by hsmart** [comment]
@Abhishekamol 
it is in src/core/utils.ts

**Comment by Ekki** [comment]
sorry don't get it... the default initialization should be disabled? really?

**Comment by Abhishekamol** [comment]
Thanks @hsmart . It is working now.
